translate.js

```diff
@@ -775,8 +855,11 @@ export default (theme) => (str) => {
                 theme.borderWidth[i[2]]
               );
               break;
+            case 'opacity':
+              out['--ow-border-opacity'] = i[2] / 100;
+              break;
             default:
-              out[`border-color`] = colorHelper(theme.colors, i[1], i[2]);
+              out[`border-color`] = `rgba(${toRgb(colorHelper(theme.colors, i[1], i[2]))},var(--ow-border-opacity,1))`;
```

Comment:
I wonder if we should have the `colorHelper` do all this wrapping in the rgba template.. so that we can go back to:

```js
out[`border-color`] = colorHelper(theme.colors, i[1], i[2]);
```

But that the helper function returns a string in the form:

```js
rgba(${toRgb(colorHelper(theme.colors, i[1], i[2]))},var(--ow-border-opacity,1))
```
Then we can just change the code in one place.

Reply:
Sure, can do that.